### PR TITLE
[Docs] add info about hardcoded translations

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | 1.11
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Inform about hardcoded keys in catalog promotion scopes/actions translations

### CODE REVIEW

File: docs/cookbook/promotions/custom-catalog-promotion-action.rst [hunk 1]
```diff
@@ -253,6 +253,14 @@ And define the translation for our new action type:
                 action:
                     fixed_price: 'Fixed price'
 
+Translations
+------------
+
+Translation keys for catalog promotion action are hardcoded to ``sylius.form.catalog_promotion.action.type`` where ``type`` is one of available catalog promotion actions:
+
+- percentage_discount
+- fixed_discount
```
Comment (CoderMaggie) [inline docs/cookbook/promotions/custom-catalog-promotion-action.rst:262]:
```suggestion
Translations for catalog promotion actions are hardcoded with such keys ``sylius.form.catalog_promotion.action.type``;
where ``type`` is one of the available catalog promotion actions: ``percentage_discount`` or ``fixed_discount``.
```

File: docs/cookbook/promotions/custom-catalog-promotion-scope.rst [hunk 2]
```diff
@@ -189,6 +189,15 @@ In our case - we have a template ``by_phrase.html.twig`` which is first before o
...
     This overriding will be suspect of change, so there won't be need for declaring ``default.html.twig`` template anymore.
 
+Translations
+------------
+
+Translation keys for catalog promotion scope are hardcoded to ``sylius.form.catalog_promotion.scope.type`` where ``type`` is one of available catalog promotion scopes:
+
+- for_products
+- for_taxons
+- for_variants
```
Comment (CoderMaggie) [inline docs/cookbook/promotions/custom-catalog-promotion-scope.rst:199]:
```suggestion
Translations for catalog promotion scopes are hardcoded with such keys ``sylius.form.catalog_promotion.scope.type``;
where ``type`` is one of the available catalog promotion actions: ``for_taxons``, ``for_products`` or ``for_variants``.
```

**Review by CoderMaggie — CHANGES_REQUESTED** [review]

File: docs/cookbook/promotions/custom-catalog-promotion-action.rst [hunk 3]
```diff
@@ -253,6 +253,12 @@ And define the translation for our new action type:
                 action:
                     fixed_price: 'Fixed price'
 
+Translations
+------------
+
+Translation keys for catalog promotion actions are hardcoded with such keys ``sylius.form.catalog_promotion.action.type``;
+where ``type`` is one of the available catalog promotion actions: ``percentage_discount`` or ``fixed_discount``.
+
```
Comment (GSadee) [inline docs/cookbook/promotions/custom-catalog-promotion-action.rst:261]: Move this info to `note` block and the content to sth like that:

```suggestion
.. note::

    There is a need to define translation key in the proper format for every catalog promotion action as they are used in form types 
    to properly display different actions: ``sylius.form.catalog_promotion.action.TYPE`` where ``TYPE`` is the catalog promotion action type.

```

File: docs/cookbook/promotions/custom-catalog-promotion-action.rst [hunk 4]
```diff
@@ -253,6 +253,10 @@ And define the translation for our new action type:
                 action:
                     fixed_price: 'Fixed price'
 
+.. note::
+    There is a need to define translation key in the proper format for every catalog promotion action as they are used in form types
+    to properly display different actions: ``sylius.form.catalog_promotion.action.TYPE`` where ``TYPE`` is the catalog promotion action type.
```
Comment (lchrusciel) [inline docs/cookbook/promotions/custom-catalog-promotion-action.rst:258]:
```suggestion
    to properly display different actions. The required type is: ``sylius.form.catalog_promotion.action.TYPE`` where ``TYPE`` is the catalog promotion action type.
```

**Review by lchrusciel — APPROVED** [review]
Thats sentence is quite long, so I would recommend split it into at least two. But I don't have strong need for it

**Review by CoderMaggie — APPROVED** [review]

**Review by Zales0123 — APPROVED** [review]

**Pull request merged**